### PR TITLE
🌐 Add Portuguese translation for `docs/pt/docs/advanced/additional-responses.md`

### PR DESCRIPTION
**Pull request by ceb10n**
Added portuguese translation to `docs/pt/docs/advanced/additional-responses.md`

As recomended it the portuguese translation discussions, I tried to translate **all terms** to portuguese, with the exception of `media type`. I looked in various translated documentation in google and I coudn't find a good translation. In general, here in Brazil we all use `media type` instead of a translation.

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit c6ed2700493b54abfa7add0a3da8a2dd4ca57941 at: https://52cf2b31.fastapitiangolo.pages.dev

File: docs/pt/docs/advanced/additional-responses.md [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+
+Você pode utilizar o mesmo parâmetro `responses` para adicionar diferentes media types para o mesmo retorno principal.
+
+Por exemplo, você pode adicionar um media type adicional de `image/png`, declarando que a sua *operação de caminho* pode retornar um objeto JSON (com o media type `application/json`) ou uma imagem PNG:
+
+```Python hl_lines="19-24  28"
+{!../../../docs_src/additional_responses/tutorial002.py!}
+```
+
+!!! note
```
Comment (Joao-Pedro-P-Holanda) [inline docs/pt/docs/advanced/additional-responses.md:175]: "Nota" also seems a good fit here

File: docs/pt/docs/advanced/additional-responses.md [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+    Note que você deve retornar a imagem utilizando um `FileResponse` diretamente.
+
+!!! info
```
Comment (Joao-Pedro-P-Holanda) [inline docs/pt/docs/advanced/additional-responses.md:178]: "Informação" can be used here.

**Review by Joao-Pedro-P-Holanda — APPROVED** [review]
Great translation :)

As a suggestion, the admonitions (text preceded by `!!!`) titles on other pages are also translated (e.g. in `docs/pt/docs/advanced/events` "tip" is translated as "Dica", "warning" as "Aviso" and "info" as "Informação").

Thanks

File: docs/pt/docs/advanced/additional-responses.md [hunk 3]
```diff
@@ -0,0 +1,240 @@
+# Retornos Adicionais no OpenAPI
+
+!!! warning
```
[the remark this answers is not in the document]
Comment (ceb10n) [reply docs/pt/docs/advanced/additional-responses.md]: Thank you @Joao-Pedro-P-Holanda , fixed it

File: docs/pt/docs/advanced/additional-responses.md [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+
+O **FastAPI** pegará este modelo, gerará o esquema JSON dele e incluirá no local correto do OpenAPI.
+
+Por exemplo, para declarar um outro retorno com o status code `404` e um modelo do Pydantic chamado `Message`, você pode escrever:
+
+```Python hl_lines="18  22"
+{!../../../docs_src/additional_responses/tutorial001.py!}
+```
+
+!!! note
```
[the remark this answers is not in the document]
Comment (ceb10n) [reply docs/pt/docs/advanced/additional-responses.md]: Thank you @Joao-Pedro-P-Holanda , fixed it

File: docs/pt/docs/advanced/additional-responses.md [hunk 5]
```diff
@@ -0,0 +1,240 @@
...
+    Lembre-se que você deve retornar o `JSONResponse` diretamente.
+
+!!! info
```
[the remark this answers is not in the document]
Comment (ceb10n) [reply docs/pt/docs/advanced/additional-responses.md]: Thank you @Joao-Pedro-P-Holanda , fixed it

File: docs/pt/docs/advanced/additional-responses.md (hunk 1 above)
Comment (ceb10n) [reply docs/pt/docs/advanced/additional-responses.md]: Thank you @Joao-Pedro-P-Holanda , fixed it

File: docs/pt/docs/advanced/additional-responses.md (hunk 2 above)
Comment (ceb10n) [reply docs/pt/docs/advanced/additional-responses.md]: Thank you @Joao-Pedro-P-Holanda , fixed it

**Comment by alejsdev** [comment]
Great. Thanks for the contribution! @ceb10n :nerd_face: 

And thanks for the help @Joao-Pedro-P-Holanda :mag: :rocket:

**Pull request merged**